test/integration/EndToEnd.t.sol

```diff
@@ -1248,4 +1249,23 @@ contract EndToEndUseCases is EndToEndFlows, VMLabeling {
 
         assertEq(s.spoke.pool(POOL_A), block.timestamp); // 2 of 2 received and processed
     }
+
+    /// forge-config: default.isolate = true
```

Comment:
Nit: I feel like we use this restriction too excessively as default. Is there any particular reason for it to be required here? Just an observation from the past reviews and since I don't have anything else to say here, I finally spoke up 😅